lib/sorted-array-index-search.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
```

Comment:
I copied the original code from `underscore` (as opposed to lodash), but yeah, this method is based on `_.sortedIndex` except that it uses comparators instead of iterators.